testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/realm/LoginSettingsTest.java

```diff
@@ -197,6 +197,34 @@ public void resetPassword() {
         assertFalse(testRealmLoginPage.form().isForgotPasswordLinkPresent());
         log.info("verified reset password is disabled");
     }
+
+
+
+    @Test
+    public void resetPasswordWithDuplicateEmailsAllowed() throws InterruptedException {
+
+        log.info("disabling email as username");
+        loginSettingsPage.navigateTo();
+        loginSettingsPage.form().setLoginWithEmailAllowed(false);
+        assertFalse(loginSettingsPage.form().isLoginWithEmailAllowed());
```

Comment:
Just one nitpick. Could you please use `org.hamcrest.MatcherAssert.assertThat` instead of `Assert.*`? I know these `Assert.*` assertions are present in the whole test class, but when a new test case is created, we want to use the `assertThat` as much as possible because it brings a lot of advantages.